File: packages/java/src/main/java/com/peruspatial/territorial/RecommendationFor.java
package com.peruspatial.territorial;

import com.peruspatial.data.TerritorialData;

public final class RecommendationFor {
  private RecommendationFor() {}

  public static String recommendationFor(String entity) {
    String found = TerritorialData.ENTITIES.get(entity);
    if (found != null) return found;
    return TerritorialData.FALLBACK.replace("{entity}", entity == null ? "" : entity);
  }
}

File: packages/rust/src/territorial/recommendation_for.rs
use crate::data::territorial::{ENTITIES, FALLBACK};

pub fn recommendation_for(entity: &str) -> String {
    for (key, value) in ENTITIES {
        if *key == entity {
            return value.to_string();
        }
    }
    FALLBACK.replace("{entity}", entity)
}
